src/data/items/resourceResearch.js

```diff
@@ -0,0 +1,127 @@
+const POINTS = {
+	rndpoints: {
+		name: "Research Points",
```

Comment:
I'm thinking that this should be the type of item that doesn't show in the inventory, much like `money`. I think inventory.js has some pointers on that but I gotta test to see how it can be done cleanly.

Reply:
probably not doing this after all
not sure how the max bank size feature would work with standard items, and research points serve their function as-is anyways